src/chain/mod.rs

```diff
@@ -39,8 +53,86 @@ pub async fn export<D: Digest>(
     seen: CidHashSet,
     skip_checksum: bool,
 ) -> anyhow::Result<Option<digest::Output<D>>, Error> {
-    let stateroot_lookup_limit = tipset.epoch() - lookup_depth;
     let roots = tipset.key().to_cids();
+    export_to_forest_car::<D>(
+        roots,
+        None,
+        db,
+        tipset,
+        lookup_depth,
+        writer,
+        seen,
+        skip_checksum,
+    )
+    .await
+}
+
+pub async fn export_v2<D: Digest>(
```

Comment:
Some documentation would be nice to have. In the current context the `export_v2` might be obvious, but a year from now someone new might struggle to understand it.

Reply:
Done.